spec/models/sponsor_spec.rb

```diff
@@ -368,6 +368,15 @@
       it 'returns only orphans that are currently sponsored' do
         expect(new_sponsor.currently_sponsored_orphans).to match_array [current_orphan]
       end
+
+      describe '#to_csv' do
+        let!(:sponsor) {build(:sponsor, name: 'John Doe', gender: 'Male', requested_orphan_count: 4, active_sponsorship_count: 1, request_fulfilled: false, country: 'FR')}
+        it 'generates csv content for given sponsors' do
+          output = "Osra Num,Name,Status,Start Date,Request Fulfilled,Sponsor Type,Country\n,John Doe,#{sponsor.status.name},#{sponsor.start_date},No (1/4),#{sponsor.sponsor_type.name},#{en_ar_country(sponsor.country)}\n"
```

Comment:
``` ruby
output= /Osra Num,Name,Status,Start Date,Request Fulfilled,Sponsor Type,Country\n,John Doe,#{Regexp.escape sponsor.status.name},#{Regexp.escape sponsor.start_date},No \(1\/4\),#{Regexp.escape sponsor.sponsor_type.name},[\"]?#{Regexp.escape en_ar_country(sponsor.country)}[\"]?\n/m
expect(output.match Sponsor.to_csv([sponsor])).to be
``` might quash that test failure without hardcoding the country, to allow that part to be tested too